### DISCUSSION TITLE
Inpainting strength

### DISCUSSION THREAD

**Discussion by Euchale** [comfyanonymous/ComfyUI#852] [Q&A] [answered]
How can I change the denoising while inpainting? I know I can change it for img2img, but sometimes I just want to change the pants but everything else looks fine, so it would be nice to be able to do img2img to just the masked part with a low denoising.

Doing it now, just makes the masked area grey.

**Comment by ltdrdata** [comment, marked as answer]
Don't use ```VAE Encode (for inpaint)```. You have to use ```Set Latent Noise```.

**Reply by Euchale** [reply]
Can you show me how I link to it properly? What goes to samples? ->
Figured it out -> Vae encode the image and use the output of that for sample.

**Reply by Renaldas111** [reply]
Can you explain me how to do this (set denoising strength for a masked part)?

**Reply by ltdrdata** [reply]
![mask](https://github.com/comfyanonymous/ComfyUI/assets/128333288/b7a14345-fd94-4501-9dd5-e09d953f2a7e)
[mask.json.txt](https://github.com/comfyanonymous/ComfyUI/files/12246631/mask.json.txt)

This way..

**Reply by ansonkao** [reply]
I'm finding that with this ComfyUI workflow, setting the denoising strength to 1.0 behaves more like a strength of 0.3 would have in Automatic1111. A denoising strength of 1.0 should essentially ignore the original image under the masked area, right? Why doesn't this workflow behave as expected?

**Reply by ansonkao** [reply]
This issue is being discussed in https://github.com/comfyanonymous/ComfyUI/issues/1186
